spec/ScoreCardSpec.js

```diff
@@ -0,0 +1,167 @@
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      expect(scorecard.frames[1].spare).toEqual(true);
+    });
+
+    it("should properly adjust the score in a frame where a spare occured", function(){
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      scorecard.tallyFrameScore();
+      expect(scorecard.frames[1].score).toEqual(15);
+    });
+
+    it("should properly adjust the score in a frame where a strike occured", function(){
+      scorecard.hitPins(10);
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      scorecard.tallyFrameScore();
+      expect(scorecard.frames[1].score).toEqual(20);
+    });
+  });
+
+  describe('10th frame scoring', function(){
+
+    it("should know that the game is not over if a stike occured in the 10th frame", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(10);
+      expect(scorecard.isGameOver).toEqual(false);
+    });
+
+    it("should know that the game is not over if two stikes occured in the 10th frame", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      expect(scorecard.isGameOver).toEqual(false);
+    });
+
+    it("should know that the game is not over if a spare occured in the 10th frame", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      expect(scorecard.isGameOver).toEqual(false);
+    });
+
+    it("should konw the game is over after three strikes in the 10th frame", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      expect(scorecard.isGameOver).toEqual(true);
+    });
+
+    it("should know the game is over after a spare in the 10th and then one extra ball is thrown", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      scorecard.hitPins(10);
+      expect(scorecard.isGameOver).toEqual(true);
+    });
+
+    it("should know the game is over after a strike in the 10th and then a spare is thrown", function(){
+      scorecard.currentFrame = 10;
+      scorecard.hitPins(10);
+      scorecard.hitPins(5);
+      scorecard.hitPins(5);
+      expect(scorecard.isGameOver).toEqual(true);
+    });
+
+    it("should know a perfect game is 300 and the game is over", function(){
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.hitPins(10);
+      scorecard.tallyFrameScore();
+      scorecard.tallyTotalScore();
+      expect(scorecard.score).toEqual(300);
+      expect(scorecard.isGameOver).toEqual(true);
+    });
+
+  });
+
+});
```

Comment:
These tests are great. However, there's always room for improvement, so:
1. Try to avoid the word "should" - it either does, or it doesn't. Prefer a more concise, descriptive style with less repeated phrases
2. One expectation per test, which you managed right up to the last test!
3. You can use helper methods to dry your tests up - rolling a perfect game could be DRYed up with a for loop, for example